extension/src/popup/locales/en/translation.json

```diff
@@ -6,6 +6,7 @@
   "A Stellar wallet for every website": "A Stellar wallet for every website",
   "About": "About",
   "Account": "Account",
+  "Account details": "Account details",
```

Comment:
I can back it out. This happens in the pre-push hook which I didn't expect to have any diff since I didn't touch any dom nodes so unclear why this happened but I can disable the hook and undo this.

Reply:
Oh weird - yeah not sure why this would trigger new translations if your branch was based off `master` 🤔